csrc/lower_validation.cpp

```diff
@@ -356,72 +356,92 @@ class VectorizeValidator : public OptInDispatch {
         vector_size,
         " however, vector sizes only upto and including 16 bytes are supported.");
 
-    auto replay_exprs = DependencyCheck::getAllExprsBetween(
-        {tv->getMaybeRFactorDomain().begin(),
-         tv->getMaybeRFactorDomain().end()},
-        {v_id});
+    auto getVectorizedIdInAllocationDomain =
```

Comment:
Can you please move this out as a separate function? I don't like long lambdas like this.

Reply:
changed